inference-engine/src/inference_engine/include/ie/ie_data.h

```diff
@@ -11,6 +11,7 @@
 
 #include <map>
 #include <memory>
+#include <ngraph/ngraph.hpp>
```

Comment:
can we include only partial shape? The whole ngraph will likely increase compilation time